docs/source/conf.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'itk-viewer-bootstrap-ui'
+copyright = '2022, Laryssa Abdala'
+author = 'Laryssa Abdala'
```

Comment:
@thewtex , I thought referred to the website's author and realize this should be under your name?

Reply:
@labdala to reflect the collaborative effort, you can put: *Insight Software Consortium*